doc/project/checklist.md

```diff
@@ -188,12 +188,12 @@ Prefix "SIM" is applicable for simulation-based DV approach only, while "FPV" is
 
 ### DV_DOC_DRAFT_COMPLETED
 
-- dv document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions, coverage objects (if applicable).
+- Dv document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions, coverage objects (if applicable).
 - Details may be missing since most items are not expected to be fully developed at this stage.
 
-### TESTPLAN_COMPLETED
+### DV_PLAN_COMPLETED
 
-A fully completed Testplan written in Hjson, indicating the list of planned tests with descriptions indicating the goal of the test and optionally details on stimulus and the checking procedure.
+A fully completed dv_plan written in Hjson, indicating the list of planned tests with descriptions indicating the goal of the test and optionally details on stimulus and the checking procedure.
```

Comment:
changing dv_plan to DV document sounds good to me, since dv_plan is very similar to testplan.
But I'm not sure if dv_plan is better than testplan, as testplan is the term widely used in industry. I think I miss something. Could you explain the rational of changing testplan to dv_plan?

Reply:
@weicaiyang @sriyerg I think we already decided this a while back. Granted the discussion was mostly between me and Sri.
But I agree that DV plan is more including that test plan alone.
and we actually use both now. - it is explained in the updated documentation that the dv plan consists of both a test plan and a functional coverage plan.

Reply:
sgtm, thanks for the explanation.